samples/drivers/led_gpio/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _gpio-led-sample:
+
+LED GPIO Sample
+###############
+
+Overview
+********
+
+The sample assumes that the LEDs are connected to GPIO lines.
+It is configured to work on boards that have defined the led0, led1, led2, led3
+aliases in their board devicetree description file.
+The LEDs are controlled, using the
+following pattern:
+
+1. Turn on LEDs
+#. Turn off LEDs
+#. Blink the LEDs
+#. Turn off LEDs
+
+Building and Running
+********************
+
+This sample can be built and flashed to a board as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/driver/led_gpio
+   :board: reel_board
+   :goals: build flash
+   :compact:
+
+After flashing the image to the board, the user LEDs on the board should act according to the given pattern.
```

Comment:
The reel_board used in your example doesn't have four LEDs on the board?

Reply:
Right I checked it. So I put there `nrf52840_pca10056` board which have aligned dts to use the LED GPIO driver. In future we can use `reel_board`.